### ISSUE TITLE
Pretrained Model "top_model.h5"

### ISSUE THREAD

**Issue by zhangqiqi1228** [robinreni96/Font_Recognition-DeepFont#9]
Could you please provide the download link for the pretrained model "top_model.h5"? Thanks!

**Comment by Pragya251990** [comment]
Did you get the link for model ?

**Comment by zhangqiqi1228** [comment]
> Did you get the link for model ?

No. But I used the codes in another project to train the model. 

You may refer to this link: https://github.com/HemingwayLee/deepfont-implement

**Comment by ogkalu2** [comment]
@zhangqiqi1228 Do you mind sharing the model you trained ?

**Comment by zhangqiqi1228** [comment]
> @zhangqiqi1228 Do you mind sharing the model you trained ?

You may try this model.

Actually, I recommend you use the codes in "deepfont-implement" project, which I shared above. I trained and inferred font detection with this project.

Model link:
https://drive.google.com/file/d/1LbGJT40OEJ9bXQpUHX9zy9XM3A-j1nCR/view?usp=sharing

**Comment by ogkalu2** [comment]
@zhangqiqi1228 Thank you ! What dataset and languages did you train on ?

**Comment by zhangqiqi1228** [comment]
> @zhangqiqi1228 Thank you ! What dataset and languages did you train on ?

I only tested 4 fonts. Just put the font file you want to train in the fonts folder. By using the create_data.py you can create your own training set. 

![image](https://github.com/robinreni96/Font_Recognition-DeepFont/assets/104071310/6fcdd543-46c4-4975-a005-7399b33dc9db)

**Comment by kelogs** [comment]
@zhangqiqi1228 

I tried the other repo as you did; recognition seems off there. Got any clues ?

https://github.com/HemingwayLee/deepfont-implement/issues/2

Thanks!
